tests/TestAppCpp/src/TestAppCpp.cpp

```diff
@@ -44,6 +51,136 @@ int main()
    return 0;
 }
 
+void TestParallel1(int argc, char** argv)
```

Comment:
Added a test function for parallel execution of the same simulation several times. 
With different parallelization options.